components/organisms/RepositoriesCart/repositories-cart.tsx

```diff
@@ -54,64 +55,69 @@ const RepositoriesCart = ({
   };
 
   return (
-    <div className="w-full lg:w-[364px] border flex flex-col gap-2 rounded-lg p-6">
-      {/* Empty state of Cart */}
-      {cartItems.length > 0 && hasItems ? (
-        cartItems.map((item, index) => (
-          <div className="flex flex-col w-full" key={`${index}/${Math.random()}`}>
-            {item}
+    <>
+      <Title className="!text-1xl !leading-none mb-4 xl:mt-10 font-medium" level={4}>
+        Added Repositories
```

Comment:
```suggestion
       Repositories Added
```
Staring at this for awhile and the the reverse just doesn't read well for me. I swapped it.

Reply:
Thanks for the comment!